.github/workflows/cloud-regress.yml

```diff
@@ -112,7 +112,7 @@ jobs:
         if: ${{ github.event.schedule && failure() }}
         uses: slackapi/slack-github-action@v1
         with:
-          channel-id: "C033QLM5P7D" # on-call-staging-stream
+          channel-id: "C06KHQVQ7U3" # on-call-qa-staging-stream
```

Comment:
Let's use `${{ vars.SLACK_ON_CALL_QA_STAGING_STREAM }}` here, then we can drop the comment

Reply:
done